iothub/device/src/IotHubBaseClient.cs

```diff
@@ -168,7 +168,7 @@ public async Task SendTelemetryAsync(TelemetryMessage message, CancellationToken
         /// <remarks>
         /// The client instance must be opened already.
         /// <para>
-        /// Use AMQP for a true batch operation. MQTT will just send the messages one after the other.
+        /// This operation is supported only over AMQP.
```

Comment:
How about, this operation is not supported over MQTT and will result in an InvalidOperationException?

Reply:
Also, document exception below.

Reply:
Added.